packages/lesswrong/lib/sql/AddFieldQuery.ts

```diff
@@ -8,16 +9,15 @@ import Table from "./Table";
  * meant for use in migrations.
  */
 class AddFieldQuery<T extends DbObject> extends Query<T> {
-  constructor(table: Table<T>, fieldName: string, skipValidation?: boolean) {
-    const fields = table.getFields();
-    const fieldType = fields[fieldName];
-    if (!skipValidation && !fieldType) {
+  constructor(table: Table<T>, fieldName: string, forceType?: Type) {
+    const fieldType = forceType ?? table.getFields()[fieldName];
+    if (!fieldType) {
       throw new Error(`Field "${fieldName}" does not exist in the schema`);
     }
     super(table, [
       "ALTER TABLE",
       table,
-      `ADD COLUMN IF NOT EXISTS "${fieldName}" ${fieldType.toString()}`,
+      `ADD COLUMN IF NOT EXISTS "${fieldName}" ${fieldType?.toString()}`,
```

Comment:
I don't think we need this change?

Reply:
Good catch! I made a change where this was needed, then I undid that change and forgot about this 🤦